src/Symfony/Component/Validator/Resources/translations/validators.es.xlf

```diff
@@ -366,6 +366,14 @@
                 <source>This value should be between {{ min }} and {{ max }}.</source>
                 <target>Este valor debe estar entre {{ min }} y {{ max }}.</target>
```

Comment:
Yes, I agree with your suggestion. Let's change `debe` by `debería` in this message.